Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersOrganisation.cshtml

```diff
@@ -224,7 +225,7 @@
                             </dt>
                             <dd class="govuk-summary-list__value">
                                 @Html.Raw(details.RegistrationDate.HasValue == true
-                                         ? $"Yes<br>On {details.RegistrationDate.Value.ToString("dd/MM/yyyy")}"
+                                         ? $"Yes<br>On {details.RegistrationDate.Value.ToString("dd MMMM yyyy")}"
```

Comment:
Same here - why not `ToFormattedString` ?

Reply:
Done